layouts/_default/list.html

```diff
@@ -1,8 +1,10 @@
 {{ partial "header.html" . }}
 <h2>{{ .Title }}</h2>
+<div id="content">
```

Comment:
This `div` isn't actually appearing in my testing for some reason on the exampleSite home page (a list of articles).

Reply:
… that's because the site home page is generated from **index.html**, not this file.